client/state/current-user/gravatar-status/actions.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import request from 'superagent';
+
+import {
+	GRAVATAR_UPLOAD_RECEIVE,
+	GRAVATAR_UPLOAD_REQUEST,
+	GRAVATAR_UPLOAD_REQUEST_SUCCESS,
+	GRAVATAR_UPLOAD_REQUEST_FAILURE
+} from 'state/action-types';
+
+export function uploadGravatar( file, bearerToken, email ) {
+	return dispatch => {
```

Comment:
Should we wrap dispatch in parens?

from [JS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#arrow-functions):

> If the function body fits on one line and there is only a single argument, feel free to omit the braces and parentheses, and use the implicit return. Otherwise, add the parentheses, braces, and use a return statement.